### PR TITLE
[SB-53] Added separated any user repositories page with avatar and username

### PR DESCRIPTION
**Pull request by easydush**

### CODE REVIEW

File: src/components/UserProfile/BaseUserProfile.tsx [hunk 1]
```diff
@@ -30,6 +37,9 @@ export const BaseUserProfile: FC<BaseUserProfileProps> = ({user, pinnedItems, lo
                         <ContributionCalendarWrapper>
                             <h2>Contribution Calendar</h2>
                             <ContributionCalendar login={user.login} calendarData={user.contributionsCollection.contributionCalendar}/>
+                            <Link onClick={handleClick}>
+                                <h3 style={{fontWeight: 'lighter'}}>See all repositories</h3>
```
Comment (MrRamka) [inline src/components/UserProfile/BaseUserProfile.tsx:41]: We can replace `h3` with styled component

File: src/components/RepositoriesList/RepositoriesList.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+import { AvatarBlock } from '../NavigationMenu/styles';
+import { UserInfoBlock } from '../UserInfoBlock';
+
+
+/**
+ * Repositories List for any user
+ * @see RepositoryList
+ */
+export const RepositoriesList: FC = () => {
+    const {username: login } = useParams();
```
Comment (MrRamka) [inline src/components/RepositoriesList/RepositoriesList.tsx]: Unusual formatting.

**Review by MrRamka — CHANGES_REQUESTED** [review]

File: src/components/UserProfile/BaseUserProfile.tsx (hunk 1 above)
Comment (easydush) [reply src/components/UserProfile/BaseUserProfile.tsx:41]: that's unnecessary

File: src/components/RepositoriesList/RepositoriesList.tsx (hunk 2 above)
Comment (easydush) [reply src/components/RepositoriesList/RepositoriesList.tsx]: i've got it  from your one

Comment (MrRamka) [reply src/components/RepositoriesList/RepositoriesList.tsx]: Where?

**Comment by MrRamka** [comment]
Better to add error handling: incorrect username

**Comment by easydush** [comment]
> Better to add error handling: incorrect username

added

**Review by MrRamka — APPROVED** [review]

**Pull request merged**